specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2022-10-01/service.json

```diff
@@ -10893,6 +10892,22 @@
       },
       "discriminator": "instanceType"
     },
+    "DisableProtectionProviderSpecificInput": {
+      "description": "Disable protection provider specific input.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProviderSpecificInput"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "instanceType": {
```

Comment:
"instanceType" is already defined in base type `ProviderSpecification`. No need to re-define